### PR TITLE
Change the content in supported data stores page

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: components/learn/platform/Platform.js [hunk 1]
```diff
@@ -282,7 +282,7 @@ export default function Platform(props) {
           </div>
 
           <div className={styles.pGroup}>
-            <h3>Ballerina persist</h3>
+            <h3>Bal persist</h3>
```
Comment (praneesha) [inline components/learn/platform/Platform.js:285]:
```suggestion
            <h3>`bal persist`</h3>
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 2]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
 API tokens for the Google sheets can be obtained using the following procedure.
-1. Get the clientID, and ClientSecret using the code following [guidelines](https://developers.google.com/identity/protocols/oauth2).
-2. Please note that you have to add https://developers.google.com/oauthplayground if you want to use OAuth 2.0 playground to receive the authorization code and obtain the access token and refresh token). The steps are as follows,
-
-    1. Go to the OAuth 2.0 playground and open the settings menu on the top right side. 
-    2. Tick `Use your own OAuth credentials` & provide the `Client ID` & `Client Secret` obtained. 
-    3. As step 1 under the  `Select & authorize APIs` section, select the required scopes for the API you are going to use. 
-    4. Proceed with the necessary Google account validation steps. 
-    5. You will obtain the Authorization code. Click `Exchange authorization code for tokens` to obtain the `Refresh token` & `Access token`.
+1. Get the clientID, and client secret using the code following [guidelines](https://developers.google.com/identity/protocols/oauth2).
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:184]:
```suggestion
1. Get the clientID and client secret using the [guidelines](https://developers.google.com/identity/protocols/oauth2).
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 3]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+2. If you want to use [OAuth 2.0 playground](https://developers.google.com/oauthplayground) to receive the authorization code and obtain the access token and refresh token, Follow below steps.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:185]:
```suggestion
2. Follow the steps below to use the [OAuth 2.0 playground](https://developers.google.com/oauthplayground) to obtain an authorization code, an access token, and a refresh token.
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 4]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   1. Go to the [OAuth 2.0 playground](https://developers.google.com/oauthplayground).
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:186]:
```suggestion
   1. Go to the [OAuth 2.0 playground](https://developers.google.com/oauthplayground).
```
Should these be sub steps?

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 5]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   2. Click the settings icon ![settings icon](https://developers.google.com/oauthplayground/assets/images/settings.png) on the top right corner of the page.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:187]:
```suggestion
   2. Click the **Settings** icon ![settings icon](https://developers.google.com/oauthplayground/assets/images/settings.png) on the top right corner of the page.
```
UI elements should be in bold and should be the same as in the UI.

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 6]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   3. Select the checkbox `Use your own OAuth credentials`.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:188]:
```suggestion
   3. Select **Use your own OAuth credentials**.
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 7]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   4. Enter the OAuth Client ID and OAuth Client secret.
+   5. Click `Close`.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:190]:
```suggestion
   5. Click **Close**.
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 8]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   6. Select the required Google Sheets API scopes.
+   7. Click `Authorize APIs`.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:192]:
```suggestion
   7. Click **Authorize APIs**.
```

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md [hunk 9]
```diff
@@ -180,12 +181,14 @@ The `script.gs` file generated from the `persist generate` command can initiate
...
+   8. Click `Exchange authorization code for tokens` to obtain the access token and refresh token.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:193]:
```suggestion
   8. Click **Exchange authorization code for tokens** to obtain the access token and refresh token.
```

File: components/learn/platform/Platform.js (hunk 1 above)
Comment (daneshk) [reply components/learn/platform/Platform.js:285]: > ```suggestion
> 
>             <h3>`bal persist`</h3>
> 
> ```

As this a header, Let's gone with Bal persist. We have the same in the BBEs as well

File: swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md (hunk 4 above)
Comment (daneshk) [reply swan-lake/learn-the-platform/ballerina-persist/supported-data-stores.md:186]: Yes

File: components/learn/platform/Platform.js (hunk 1 above)
Comment (praneesha) [reply components/learn/platform/Platform.js:285]: Still, we need backticks.

**Review by praneesha — APPROVED** [review]

**Pull request merged**